lib/snyk-to-html.js

```diff
@@ -36,10 +37,52 @@ function groupVulns(vulns) {
   return result;
 }
 
+function compileTemplate(fileName) {
+  const templ = fs.readFileSync(fileName, 'utf8');
+  return Handlebars.compile(templ);
+}
+
 function generateTemplate(data, template) {
   data.vulnerabilities = groupVulns(data.vulnerabilities);
-  var htmlTemplate = fs.readFileSync(template, 'utf8');
-  return Handlebars.compile(htmlTemplate)(data);
+  const cssFile = path.join(
+    path.dirname(template),
+    'test-report.inline-css.hbs'
+    );
+  Handlebars.registerPartial('inline-css', compileTemplate(cssFile));
+  var htmlTemplate = compileTemplate(template);
+  return htmlTemplate(data);
+}
+
+function mergeData(dataArray) {
+  const vulnsArrays = dataArray.map(d => d.vulnerabilities);
+  const aggregateVulnerabilities = [].concat(...vulnsArrays);
```

Comment:
This is not node4 compatible I believe. Just drop the `...` and it’ll be ok?

Reply:
Literally no idea - @miiila ?
(Also note that I'll have this under typescript later today as well)

Reply:
@kevinwright Unfortunately, @adrukh is right, sorry for tricking you into this. It's available from Node 5 😢